docs/API/core/app-actions.md

```diff
@@ -326,6 +326,42 @@ Fliplet.App.Actions.remove('confirm-booking').then(function () {
 
 ---
 
+## Testing and debugging an action
+
+Once you have created an app action and added the code you want to run to the onRemoteExecution() function you should test it and see if it running as expected. Note that you cannot use developer tools to view your console.log inside the onRemoteExecution() function as this is running on the server side. Here is the general steps to test an app action: 
```

Comment:
Wrap function names and variables with these carets:
```
`onRemoteExecution()`
```

Reply:
@theexperimenter please resolve conversations when new commits have been pushed to address the comments.

Reply:
Agree with this change.